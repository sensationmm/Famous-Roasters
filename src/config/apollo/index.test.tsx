import { ApolloProvider } from '@apollo/client'
import { render, waitFor } from '@testing-library/react'
import React from 'react'

import { storeFrontClient } from './index'

describe('Apollo client', () => {
  it('Renders correctly', async () => {
    const { container } = render(
      <ApolloProvider client={storeFrontClient()}>
        <span />
      </ApolloProvider>,
    )
    await waitFor(() => new Promise((res) => setTimeout(res, 0)))
    expect(container).toMatchSnapshot()
  })
})
