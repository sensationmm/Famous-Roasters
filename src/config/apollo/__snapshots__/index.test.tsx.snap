// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Apollo clients Renders famousRoastersClient correctly 1`] = `
<div>
  <span />
</div>
`;

exports[`Apollo clients Renders hygraphClient correctly 1`] = `
<div>
  <span />
</div>
`;

exports[`Apollo clients Renders storefrontClient correctly 1`] = `
<div>
  <span />
</div>
`;
